### PR TITLE
CME changes

### PR DESCRIPTION
**Pull request by Gandalf2k15**
## About The Pull Request

Bubbles give 5 seconds more time to neutralize. 
Only extreme + CME's actually cause a fire when neutralized.
All obnoxious sounds have been removed.
Chance to spawn has been cut in half.
You can now neutralize an anomaly using analyzers.
CME's are now more likely to drop anomaly cores.

closes https://github.com/Skyrat-SS13/Skyrat-tg/pull/7816

## Changelog
:cl:
balance: CME's have yet again been reworked and are now more rewarding and easier to deal with.
/:cl:

### CODE REVIEW

**Comment by CameronWoof** [comment]
God, no. These events were disabled for a reason. If the frequency of events we have now is too high, decrease the rate at which random events are called. We **do not** need any of these back. 

**These were disabled for good reason. Do not bring them back.**

**Comment by BurgerLUA** [comment]
Why in god's green earth are you grouping a nerf that everyone wants with an event re-add that will be extremely controversial? This goes against modularization standards. https://github.com/Skyrat-SS13/Skyrat-tg/blob/master/modular_skyrat/readme.md

**Comment by Gandalf2k15** [comment]
We'll actually be doing this differently, but before you explode, we need to discuss it first. DNM for now.

**Comment by Gandalf2k15** [comment]
RTM

File: modular_skyrat/modules/cme/code/cme.dm [hunk 1]
```diff
@@ -266,11 +261,7 @@ Armageddon is truly going to fuck the station, use it sparingly.
 	empulse(src, pulse_range_heavy, pulse_range_light)
 	explosion(src, 0, 3, 10, flame_range = 10)
 	playsound(src,'sound/weapons/resonator_blast.ogg',100,TRUE)
-	for(var/i in GLOB.mob_list)
-		var/mob/M = i
-		if(M.client && M.z == z)
-			SEND_SOUND(M, sound('modular_skyrat/modules/cme/sound/cme.ogg'))
-			shake_camera(M, 15, 1)
+	playsound(src,'modular_skyrat/modules/cme/sound/cme.ogg', 100)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/cme/code/cme.dm:264]: Finally 🙏

**Review by GoldenAlpharex — APPROVED** [review]
This looks good, we can always adjust the time they take to pop further, later down the line.

**Pull request merged**